lte/gateway/c/session_manager/test/test_set_session_manager_handler.cpp

```diff
@@ -11,52 +11,68 @@
  * limitations under the License.
  */
 
+#include <memory>
+
 #include <folly/io/async/EventBaseManager.h>
 #include <glog/logging.h>
 #include <gtest/gtest.h>
 
-#include <memory>
-
 /* session_manager.grpc.pb.h and SessionStateEnforcer.h
  * included in "SetMessageManagerHandler.h"
  */
-#include "AmfServiceClient.h"
-#include "magma_logging.h"
+#include "SetMessageManagerHandler.h"
 #include "includes/MagmaService.h"
-#include "PipelinedClient.h"
 #include "ProtobufCreators.h"
 #include "RuleStore.h"
 #include "includes/ServiceRegistrySingleton.h"
-#include "SessiondMocks.h"
 #include "SessionState.h"
 #include "SessionStore.h"
-#include "SetMessageManagerHandler.h"
+#include "SessiondMocks.h"
 #include "StoredState.h"
+#include "magma_logging.h"
+#include "PipelinedClient.h"
+#include "AmfServiceClient.h"
+#include "Consts.h"
 
+using grpc::ServerContext;
+using grpc::Status;
+using ::testing::InSequence;
 using ::testing::Test;
 
+#define SESSIOND_SERVICE "sessiond"
+#define SESSIOND_VERSION "1.0"
+
 namespace magma {
 
 class SessionManagerHandlerTest : public ::testing::Test {
  public:
   virtual void SetUp() {
-    rule_store    = std::make_shared<StaticRuleStore>();
+    rule_store_   = std::make_shared<StaticRuleStore>();
     session_store = std::make_shared<SessionStore>(
-        rule_store, std::make_shared<MeteringReporter>());
-    auto pipelined_client = std::make_shared<magma::AsyncPipelinedClient>();
-    amf_srv_client        = std::make_shared<magma::AsyncAmfServiceClient>();
-
+        rule_store_, std::make_shared<MeteringReporter>());
+    std::unordered_multimap<std::string, uint32_t> pdr_map;
+    pipelined_client = std::make_shared<MockPipelinedClient>();
+    amf_srv_client   = std::make_shared<magma::AsyncAmfServiceClient>();
     magma::mconfig::SessionD mconfig;
     mconfig.set_log_level(magma::orc8r::LogLevel::INFO);
     mconfig.set_relay_enabled(false);
 
+    auto config =
```

Comment:
I think we should mock out this config loading step. Can we manually construct the config value we want here instead of loading it from the machine?

Reply:
Yes. defined the value here instead of fetching from config.